src/ExtEventLoop.php

```diff
@@ -178,8 +178,7 @@ public function tick()
 
         $this->futureTickQueue->tick();
 
-        // @-suppression: https://github.com/reactphp/react/pull/234#discussion-diff-7759616R226
-        @$this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
+        $this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
```

Comment:
I've did some tests today but i can't reproduce this. The comment in https://github.com/reactphp/react/pull/234#discussion-diff-7759616R226 is also not clear in **what** exactly produces these warnings. Maybe @nrk or @jmalloc can shed some light on this.

I agree that the `@` is really bad here because it also supresses any warnings and errors from the callbacks.